# Enum values for the ConductorState message
uint8 UPCAST = 0
uint8 DOWNCAST = 1
uint8 TRANSIT_TO_SCHEDULED_DEPTH = 2
uint8 TRANSIT_TO_PEAK_DEPTH = 3
uint8 WAIT_FOR_IFCB = 4
uint8 IFCB_BEADS = 5
uint8 IFCB_BIOCIDE = 6
uint8 IFCB_BLEACH = 7
uint8 IFCB_DEBUBBLE = 8
uint8 IFCB_RUNSAMPLE = 9
uint8 IFCB_CARTRIDGE_DEBUBBLE = 10
uint8 IFCB_PRIMESAMPLETUBE = 11